app/src/main/java/org/supla/android/listview/ChannelListView.java

```diff
@@ -448,24 +466,12 @@ public boolean onTouchEvent(MotionEvent ev) {
 
                     if (channelLayout != null) {
                         if (channelLayout.getDetailSliderEnabled()) {
-                            Object obj = getItemAtPosition(pointToPosition((int) X, (int) Y));
-
-                            if (obj instanceof Cursor) {
-                                ChannelBase cbase;
-
-                                if (((ListViewCursorAdapter) getAdapter()).isGroup()) {
-                                    cbase = new ChannelGroup();
-                                } else {
-                                    cbase = new Channel();
-                                }
+                            ChannelBase cbase = getChannelBaseAtXY(X, Y);
+                            if(cbase != null && getDetailLayout(cbase) != null) {
```

Comment:
Would be better if we'll stay in one code formatting. Could please format your changes, because I can see here three different ways, how the if statement is formatted. My proposal is to always use following:
```
if (argument) {

} else {

}
```

Reply:
I agree